build_debian.sh

```diff
@@ -37,6 +37,7 @@ LINUX_KERNEL_VERSION=5.10.0-8-2
 ## Working directory to prepare the file system
 FILESYSTEM_ROOT=./fsroot
 PLATFORM_DIR=platform
+PLATFORM_SECURITY_CONF=./.sbsign.conf
```

Comment:
Can we change to simply export it as an environment variable in slave.mk?

Reply:
Fixed, Now variable can be passed as below

   Add environmental variable at rules/config.user
   Ex:
   SONIC_ENABLE_SECUREBOOT_SIGNATURE := y
   SIGNING_KEY := /path/to/key/private.key
   SIGNING_CERT := /path/to/public/public.cert